constraint/pkg/client/drivers/local/driver.go

```diff
@@ -72,6 +73,20 @@ type Driver struct {
 	clientCertWatcher *certwatcher.CertWatcher
 }
 
+// regoResultMeta has rego specific metadata for a result.
+type regoResultMeta struct {
+	// templateRunTime is the number of milliseconds it took to evaluate all constraints for a template.
+	templateRunTime float64
+	// engineType is an engine set string for the kind of underlying engine that was used for a client.Review() call.
+	engineType string
+	// constraintCount indicates how many constraints were evaluated for an underlying engine eval call.
+	constraintCount uint
+}
+
+func (rm *regoResultMeta) EngineStatsString() string {
+	return fmt.Sprintf("totalTemplateRuntime: %.4f, engineType: %s, constraintCount: %d", rm.templateRunTime, rm.engineType, rm.constraintCount)
```

Comment:
If engine type is important for analysis, we may want to have that be reported by the framework to guarantee uniformity of how its reported. For now it can be omitted.